docs/pages/deploy-a-cluster/deployments/aws-terraform.mdx

```diff
@@ -295,12 +295,13 @@ $ terraform import aws_acm_certificate.cert <certificate_arn>
 
 ### Instances
 
-Our reference deployment will provision the following instances for your cluster by default:
+Our reference deployment will provision the following instances for your cluster by default with AWS Auto Scaling Groups (ASG):
 
 - 2 x m4.large Teleport **auth** instances in an ASG, behind an internal network load balancer, configured using DynamoDB for
   shared storage. [The desired size of the ASG is configured here](https://github.com/gravitational/teleport/blob/master/examples/aws/terraform/ha-autoscale-cluster/auth_asg.tf#L11)
 - 2 x m4.large Teleport **proxy** instances in an ASG, behind a public-facing load balancer - NLB for LetsEncrypt, ALB for ACM. [The desired size of the ASG is configured here](https://github.com/gravitational/teleport/blob/master/examples/aws/terraform/ha-autoscale-cluster/proxy_asg.tf#L12)
-- 1 x m4.large Teleport **node** instance in an ASG. [The desired size of the ASG is configured here](https://github.com/gravitational/teleport/blob/master/examples/aws/terraform/ha-autoscale-cluster/node_asg.tf#L10)
+- 1 x m4.large Teleport **node** instance in an ASG. Server instances in this ASG will join with Server SSH Access enabled and connected to the **auth**
```

Comment:
I'm not sure I've seen us use the term _Server SSH Access_ before. Don't we usually just say _Server Access_ or _with Teleport's SSH service enabled_?

Reply:
update to Teleport SSH Service